locale/en/hammer_cli_foreman_scc_manager.po

```diff
@@ -0,0 +1,111 @@
+# English translations for hammer_cli_foreman_scc_manager package.
+# Copyright (C) 2023 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the hammer_cli_foreman_scc_manager package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: hammer_cli_foreman_scc_manager 0.1.0\n"
```

Comment:
Same here about the version number. It is 0.2.1 and I want to bump a new version today such that I can update the packaging.

Reply:
I checked in other projects and it does not seem to be in line with the actual plugin version... So let's just ignore it for now.